### DISCUSSION TITLE
Distinguishable primary and secondary cursor colors

### DISCUSSION THREAD

**Discussion by peter-jerry-ye** [helix-editor/helix#4833] [Q&A] [answered]
Hi,

I'm walking through the tutor today, and for the multi cursor mode part, I found that when using `(` `)` jumping between the selection, I can't tell the difference about which is currently focused on. Is just me or there's actually no color difference?

I'm using the default theme on Manjaro with the latest version (22.08).

**Comment by the-mikedavis** [comment]
The matching bracket is currently only highlighted for the primary cursor: https://github.com/helix-editor/helix/issues/3228

**Comment by the-mikedavis** [comment, marked as answer]
The default theme also doesn't distinguish between primary and secondary cursor colors: it only has a `ui.cursor` scope defined which is used for both. You can use the `ui.cursor.primary` theme key to color the primary cursor and `ui.cursor` for any secondary cursors

**Reply by peter-jerry-ye** [reply]
Ok, thank you for your response. I also edited the title to clarify my question.

**Comment by archseer** [comment]
We should add more primary cursor styles to themes: https://github.com/helix-editor/helix/issues/1833

I experimented with this a bit for the default theme but couldn't decide yet on a reasonable value.
